### PR TITLE
Add <p> example

### PR DESCRIPTION
**Pull request by Elchi3**
Example for https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p

Nothing really exciting to see here. The idea is that you might want to learn about the differences between `<p>` and `<br>` so I've shown a paragraph with two sentences. Then one with the sentences separated by a `<br>` and then another where the two sentences are both paragraphs. Does this make sense?

### CODE REVIEW

**Comment by schalkneethling** [comment]
@Elchi3 the options are indeed limited here. I wonder if it would aid visually if you demarcate the paragraphs? Something like:

```
padding: 5px;
border: 1px solid #999;
```

File: live-examples/html-examples/text-content/p.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+Maecenas viverra facilisis rutrum</p>
+
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. <br>
```
Comment (schalkneethling) [inline live-examples/html-examples/text-content/p.html]: Nit: Code style: This is not documented yet but, I prefer that we explicitly close self closing tags. Even though it is not required in HTML5. Thanks

**Comment by dagolinuxoid** [comment]
The snippet was merged 12 days ago, I'm just wondering — when it's be available on mdn docs site?

**Comment by wbamberg** [comment]
We're don't want to add them to MDN pages until https://github.com/mdn/interactive-examples/issues/860 is fixed (and there are possibly more blockers, but that's the main one). They are live under https://interactive-examples.mdn.mozilla.net/ (e.g. https://interactive-examples.mdn.mozilla.net/pages/tabbed/p.html) but that's just the hosting site from which we will embed them.

But most of the work to add examples is writing them, so we're getting on with that while we are blocked. Once we are unblocked, adding them to the pages will be really quick.

**Pull request merged**